src/pages/careers/_join_your_team.js

```diff
@@ -0,0 +1,219 @@
+/* eslint-disable no-unused-vars */
+import React from 'react'
+import styled, { css } from 'styled-components'
+import { graphql, useStaticQuery } from 'gatsby'
+import TeamCard from './_layout-components/_team-card'
+import { Container, CssGrid, SectionContainer } from 'components/containers'
+import { Text, Header } from 'components/elements'
+import device from 'themes/device'
+
+const DiffHeader = styled(Header)`
+    margin-bottom: 5rem;
+    font-size: 48px;
+    line-height: 60px;
+
+    @media ${device.tablet} {
+        font-size: 28px;
+        line-height: 34px;
+    }
```

Comment:
```suggestion
```

These sizing rule can be omitted. We already have handling for this set of style. You can set it by passing the prop `type="heading-2"` to the `Header` component. It should handle for everything including for mobile 🙏🏼